core/modules/commands/savelibrarytiddlers.js

```diff
@@ -77,6 +77,9 @@ Command.prototype.execute = function() {
 			iconText = iconTiddler.text,
 			iconContent;
 		if(iconType && iconText) {
+			if (iconType === "image/svg+xml") {
+				iconText = iconText.replace("<svg ", "<svg xmlns=\"http://www.w3.org/2000/svg\" ")
```

Comment:
Why is this needed?

Reply:
There is a super tricky problem: 

In line 79 there is `if(iconType && iconText) {` ... which tests the icon-tiddler for the `iconType`. .. BUT most TW icons don't have a `type` field, because we don't need it. .. So the plugin icon will _never ever_ be used in the "add plugins" modal dialogue. 

The attempt to fix this was: I did add a type field to my icons. ... BUT ... If icons have a type-field the existing "plugins tab" in the control panel doesn't show the icon anymore. 

So the "fix above" won't work anyway. ... I'll need to create a completely separate Issue for this one. 

----------

Adding a type field to svg-icons doesn't fix the problem. SVG's need to have the `xmlns=\"http://www.w3.org/2000/svg\"` attribute to `iconContent = $tw.utils.makeDataUri(iconText,iconType);` work as desired.  .... BUT ... I don't want to add this attribute. 

SO we have a "chicken / egg" problem here. That's why we need a new **independent issue**

Reply:
see: https://github.com/Jermolene/TiddlyWiki5/pull/5418/commits/1c6e6e8f4a1d05aa2ae3dad97ce7ff3520cef849